### PR TITLE
Add Dashboard Deploy Script and Add Rule to Database Deploy Script

### PR DESCRIPTION
**Pull request by mcdonnnj**
# 

## 🗣 Description
This PR adds a script to facilitate deploying a new CyHy Dashboard AMI. There is also the addition of a rule that is required by the dashboard to the database deploy script.

The database deploy script also had its spacing adjusted to mirror the other deploy scripts in this repository.

## 💭 Motivation and Context
I ran into an issue with deployment today that involves the dashboard and it is one of the images that lacks a deploy script. Having a targeted deploy script is convenient when working with these instances.

The rule added to the database deploy script covers allowing the dashboard and reporter instances ingress to the database instance for a MongoDB connection.

I am uncertain if security group to security group rules need to be targeted when doing a redeployment. However, other deploy scripts have them, so I have included all relevant ones in the dashboard deploy script.

## 🧪 Testing
I used the dashboard script to do a targeted deploy after a forced instance replacement. I was able to successfully use dashboard services.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by felddy — APPROVED** [review]
You fight for the Users! 
💾 Pew pew!

**Review by jsf9k — APPROVED** [review]
Excellent addition!

File: terraform/deploy_new_dashboard_ami.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# deploy_new_database_ami.sh region workspace_name
```
Comment (dav3r) [inline terraform/deploy_new_dashboard_ami.sh:3]: I'm late to the review game here, but `deploy_new_database_ami` -> `deploy_new_dashboard_ami`

Comment (mcdonnnj) [reply terraform/deploy_new_dashboard_ami.sh:3]: Ack, I should've done a find/replace. The mistakes we make trying to get something working in a hurry.

**Pull request merged**